validate.go

```diff
@@ -0,0 +1,42 @@
+package huh
+
+import (
+	"fmt"
+	"unicode/utf8"
+)
+
+// ValidateNotEmpty checks if the input is not empty.
+func ValidateNotEmpty() func(s string) error {
+	return func(s string) error {
+		if s == "" {
+			return fmt.Errorf("input cannot be empty")
```

Comment:
I think it would be nice to export these errors, so users can use `errors.Is` with them 🤔

Reply:
Yes make sense.

I was also thinking that error message can be a input. Because the error message might depend on final use case.